news/2024/2024-07-23-osu-austria-conventions-2024.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: osu! Austria @ Conventions 2024
+date: 2024-07-23 12:00:00 +0000
+---
+
+The osu! Austria community proudly presents its appearance at not one, but two upcoming conventions in 2024!
+
+![](/wiki/shared/news/2024-07-23-osu-austria-conventions-2024/Banner.png)
+
+Next to osu! booths, there will also be LAN tournaments, in which you can show off your skills and win some nice goodies. What are you waiting for? Get all the information you need down below!
+
+## AniNite 2024
+
+![](/wiki/shared/news/2024-07-23-osu-austria-conventions-2024/aninite.png)
+
+Join us at the biggest anime and manga convention in Austria, [AniNite](https://www.aninite.at/en/), which has over 25,000 yearly visitors. You can find us from **August 9th to 11th** at our osu! booth, where there will be 12 PCs for you to play on and show everyone what you're made of.
+
+During this time, we'll host a **battle royale–styled** osu! tournament. Make sure to register during the event and take home the trophy!
+
+## YuniCon 2024
+
+![](/wiki/shared/news/2024-07-23-osu-austria-conventions-2024/yunicon.png)
+
+Next up, we're setting up an osu! booth at [Yunicon 2024](https://www.yunicon.at/en/). During all 4 days of the convention, you'll be able to meet our community, ask questions and play on 10 pre-organised computers!
+
+On top of that, you'll also be able to witness the biggest osu! 1v1 LAN tournament with a single-elimination bracket that Austria has ever seen!
+
+You'll be able to sign-up for the tournament online. Please join our [Discord server](https://discord.com/invite/dNFuHNmZF4) to get notified when sign-ups open.
+
+Qualifiers will be played out online as well and scheduling your lobby will be handled over at our *Discord server*. After you qualified for the Round of 16, **you're required to show up**. A no-show will automatically **eliminate** you from the tournament and open your spot for the next possible seed that is present at the event. Bring the peripherals you're comfortable with and prepare to bash your opponents!
+
+The top 2 players of the tournament will ferociously duke it out for some handsome physical prizes and a bespoke profile badge on the main stage of the event, and the top 16 players won't go home empty-handed either as they will receive some nice goodies as well!
```

Comment:
I assume the physical prizes will only go to the winner? rewrote this the way i liked it more, but its subjective ofc
```suggestion
The top 2 players of the tournament will ferociously duke it out on the main stage of the event, with the winner receiving handsome physical prizes and a bespoke profile badge. The top 16 players won't go home empty-handed either, as they will receive some nice goodies as well!
```